### DISCUSSION TITLE
new to omp, want to enable svn type

### DISCUSSION THREAD

**Discussion by johandanforth** [JanDeDobbeleer/oh-my-posh#4350] [Troubleshoot]
I saw in the docs that there is some subversion support included, so I added the sample configuration to the json file I'm using. After restarting powershell terminal I was hoping to get some svn-info on the prompt, but alas no... I'm sure I did it wrong, I just added the configuration for type svn next to the git configuration in the json file. 

Thanks for any help!

**Comment by JanDeDobbeleer** [comment]
@johandanforth care to share your configuration?

**Reply by johandanforth** [reply]
Hard, I'm in an disconnected environment on my work computer :D But in principal, I'm using the jandedobbeleer.omp.json config, so I took that json, found the "git" segment section and added this next to it:

``` json
{
  "type": "svn",
  "style": "powerline",
  "powerline_symbol": "",
  "foreground": "#193549",
  "background": "#ffeb3b",
  "properties": {
    "fetch_status": true
  }
}

```

That's about it, not sure if I need to do anything else, except for running the init again. Now, we're about to move to git from subversion soon, so this problem is about to go away, but until then, it would be helpful :)

Thanks for replying, much appreciated, love the tool!

**Reply by johandanforth** [reply]
Hold your horses! I found the reason! I had the themes in multiple places, and was of course editing the wrong file! *hit self* Works perfectly now!
